src/diffusers/models/attention_processor.py

```diff
@@ -2924,6 +2924,232 @@ def __call__(
         return hidden_states
 
 
+class PAGIdentitySelfAttnProcessor2_0:
+    r"""
+    Processor for implementing scaled dot-product attention (enabled by default if you're using PyTorch 2.0).
```

Comment:
We are not implementing SDPA here. 

```suggestion
    Processor for implementing PAG using scaled dot-product attention (enabled by default if you're using PyTorch 2.0).
    
    PAG reference: https://arxiv.org/abs/2403.17377
```